src/app/page/Signup/Signup.js

```diff
@@ -7,31 +7,70 @@ import Button from "../../component/Button";
 import { FormWrapper, HeaderText, DescriptionText, PaddedDiv } from "./Signup.style";
 
 const SignupPage = () => {
+  const [form, setForm] = useState({
```

Comment:
There is a useForm hook inside the hooks folder that basically do the same thing
Can we use that for reusability? Thanks

Reply:
Can you point me to an article or somewhere I can read more about this? I never used hooks so  I don't quite understand your approach. Sorry.

Reply:
Sure, there is a hook directory which contains useForm hook
It exposes the handleChange, setValues and values
You can use that instead of creating your own hook

Reply:
Man, that is awesome. I had no idea you can do something like that. Explains why I completely missed the hooks directory. Thanks for pointing that out.